bin/internal/goldens.version

```diff
@@ -1 +1,5 @@
+/// Deprecated, See:
+/// https://github.com/flutter/flutter/wiki/Writing-a-golden-file-test-for-package:flutter
```

Comment:
Will update this once there has been some feedback on the workflow proposed here.